src/refmterr/lib/stylish.re

```diff
@@ -97,6 +97,8 @@ module type StylishSig = {
     string;
 };
 
+Pastel.(setMode(Terminal));
```

Comment:
I don't think we want this here, we probably want to be setting the environment variable FORCE_COLOR=true or using Pastel.useMode(Terminal) in our tests. Alternatively we could use Pastel.useMode(HumanReadable) to generate more human readable snapshots (e.g. we get <Red> </Red> instead of [031m/...). 

This change here causes the ansi escape sequences to always be generated for everyone, which leads to a bad experience if what they are using doesn't support them

Reply:
Would make sense to set `HumanReadable` instead without doing the snapshots?

Reply:
I would move that to the test and update all the expected.txt "snapshots".